packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -213,19 +231,23 @@ public void onResume(@NonNull LifecycleOwner owner) {
     onActivityResumed(null);
   }
 
+  // hack: dialog opens twice on Android 8
```

Comment:
Do we know why? Do the handlers below fire twice for some reason?

Reply:
I am not sure. This happened a while ago and don't remember the specifics and I have not been able to reproduce the issue in emulators today.

Reply:
Let's remove it if it cannot be reproduced.

Reply:
removed